src/scripts/hubot-jenkins-notifier.coffee

```diff
@@ -117,22 +140,50 @@ class JenkinsNotifier
     if !data || typeof data.build != 'object'
       @error new Error("Unable to process data"), req.body
       return
+
+    @logMessage(query.trace, data.build)
+
     fullurl = data.build.full_url || data.build.url
 
+    if data.build.phase in ['COMPLETED']
+      @logMessage(query.trace, "Ignoring phase COMPLETED")
+      return
+
+    if data.build.phase in ['STARTED']
+      @logMessage(query.trace, "#{data.name} #{data.build.phase} #{data.build.status}")
+
+      @robot.send envelope, "#{data.name} build ##{data.build.number} started: #{fullurl}"
```

Comment:
Hm, I am not so sure about this... As far as I understand `shouldNotify` is used to determine a state change. But for STARTED there is no state change, right? So I figured, if somebody turns on STARTED they probably want to be notified in any case.
But if you want to we can still wrap it in `shouldNotify`